functions/go/set-enforcement-action/README.md

```diff
@@ -0,0 +1,42 @@
+# set-enforcement-action
+
+## Overview
+
+<!--mdtogo:Short-->
+
+Applies the supplied enforcement action on [policy constraints](https://cloud.google.com/anthos-config-management/docs/concepts/policy-controller#constraints) within a package.
+
+<!--mdtogo-->
+
+Policy Controller [constraint objects](https://cloud.google.com/anthos-config-management/docs/how-to/creating-constraints) enable you to enforce policies for your Kubernetes clusters.
```

Comment:
May want to reword it to:
`Policy Controller allows users to define [constraints](https://cloud.google.com/anthos-config-management/docs/how-to/creating-constraints) to enforce policies for your Kubernetes clusters.`

Reply:
Done.